erpnext/accounts/doctype/account/account_tree.js

```diff
@@ -56,7 +56,7 @@ frappe.treeview_settings["Account"] = {
 	toolbar: [
 		{
 			condition: function(node) {
-				return !node.root && frappe.boot.user.can_read.indexOf("GL Entry") !== -1
```

Comment:
Can I add Account permission, View Ledger button only display if user has permission to read account data